### ISSUE TITLE
Temperature reporting not working

### ISSUE THREAD

**Issue by stephantamminga** [dougforpres/NINASonyCameraPlugin#1]
The sensor temperature is written to the filename, but is not displayed on the Equipment tab or camera info in the imaging tab.

Sony A7 III

**Comment by dougforpres** [comment]
Yeah, the temperature feature doesn't work as the camera (or at least the only camera I have access to, the a6400) does not provide a property telling me the temperature.
There is a hack I added that only works for the ASCOM (non-Nina plugin) version where it tries to pull the temperature from the most recent image (as it's part of the exif data) - but I wouldn't rely on that.
